frida_navigation_interfaces/srv/CreateGoal.srv

```diff
@@ -0,0 +1,4 @@
+geometry_msgs/PoseStamped target
+std_msgs/Float64 offset
```

Comment:
For use in external scripts add more description to the offset and its direction